source/renderer/app/components/wallet/WalletRestoreDialog.scss

```diff
@@ -15,6 +15,20 @@
     margin-bottom: 20px;
   }
 
+  .newLabel {
+    background-color: var(--theme-main-body-background-color);
+    border-radius: 3px;
+    color: var(--theme-main-body-messages-color);
```

Comment:
@DeeJayElly please add separate CSS variable for this color

Reply:
Added